### PR TITLE
[tasks] Update type of `ModelData.base_model`

### PR DESCRIPTION
**Pull request by julien-c**
needed for internal PR cc @davanstrien

### CODE REVIEW

**Review by Wauplin — APPROVED** [review]
Looks good.

I just realized `base_model` is not a supported field by default in `huggingface_hub.ModelCardData`. Will open a PR soon.

File: packages/tasks/src/library-ui-elements.ts [hunk 1]
```diff
@@ -72,7 +72,7 @@ model = BaseModel.from_pretrained("${model.id}")`,
 ];
 
 function get_base_diffusers_model(model: ModelData): string {
-	return model.cardData?.base_model ?? "fill-in-base-model";
+	return model.cardData?.base_model?.toString() ?? "fill-in-base-model";
```
Comment (osanseviero) [inline packages/tasks/src/library-ui-elements.ts:75]: I guess it's ok to assume we only have 1 base model for diffusion models

**Review by osanseviero — APPROVED** [review]

File: packages/tasks/src/library-ui-elements.ts (hunk 1 above)
Comment (julien-c) [reply packages/tasks/src/library-ui-elements.ts:75]: yes, i hesitated with taking the 1st element but 🤷

honestly we should have specified / enforced an array of string from the beginning, it's always kinda messy to do it after the fact

**Comment by julien-c** [comment]
@coyotte508 any way to do a release? 🙏

**Comment by julien-c** [comment]
can you update the npm requirement on my moon-landing PR when it's released? thanks a lot!

**Pull request merged**